rom/laghos.cpp

```diff
@@ -990,6 +1105,7 @@ int main(int argc, char *argv[])
     // defines the Mult() method that is used by the time integrators.
     if (!rom_online) ode_solver->Init(*oper);
     if (fom_data) oper->ResetTimeStepEstimate();
+    if (rom_build_database) cout << "Blast energy factor: " << romOptions.blast_energyFactor << endl;
```

Comment:
This assumes the database parameter is the energy factor, right? If this cannot be generalized, maybe it is better to remove it.